docs/upgrading.md

```diff
@@ -0,0 +1,66 @@
+# Upgrading Contour to 0.15
+
+## Changes in 0.15
+
+### Breaking change
+The major breaking change in Contour 0.15 is that `contour serve` (the mode that actually serves the control plane for Envoy) now requires that either TLS certificates be available, or you supply the `--insecure` parameter.
+
+**Without one of these options, Contour will not start.**
```

Comment:
**If you do not supply TLS details or `--insecure` `contour serve` will not start.**